cpp/include/ray/api/generated/exec_funcs.generated.h

```diff
@@ -50,96 +50,29 @@ std::shared_ptr<msgpack::sbuffer> ExecuteActorFunction(
   return std::make_shared<msgpack::sbuffer>(Serializer::Serialize(return_value));
 }
 
-// 0 args
-template <typename ReturnType>
+template <typename ReturnType, typename... Args>
 std::shared_ptr<msgpack::sbuffer> NormalExecFunction(
     uintptr_t base_addr, size_t func_offset,
     const std::vector<std::shared_ptr<RayObject>> &args_buffer) {
-  return ExecuteNormalFunction<ReturnType, ReturnType>(
-      base_addr, func_offset, args_buffer, TaskType::NORMAL_TASK);
+  return ExecuteNormalFunction<ReturnType, ReturnType, Args...>(
+      base_addr, func_offset, args_buffer, TaskType::NORMAL_TASK,
+      std::shared_ptr<Args>{}...);
```

Comment:
Good point. And can you modify the file path and file name from `cpp/include/ray/api/generated/exec_funcs.generated.h` to `cpp/include/ray/api/exec_funcs.generated.h`?

Reply:
And you can delete the description in the head of this file:

/// This file is auto-generated. DO NOT EDIT.

// TODO(Guyang Song): code generation

Reply:
No problem.